### PR TITLE
adding timestamp token to tc-github template

### PR DESCRIPTION
**Pull request by garbas**

### CODE REVIEW

**Comment by garbas** [comment]
Feel free to push changes to my fork and update this commit

**Review by djmitche — CHANGES_REQUESTED** [review]
Can you add mention of it to `docs/usage.md`?

**Comment by garbas** [comment]
@djmitche docs updated.

File: src/intree.js [hunk 1]
```diff
@@ -88,6 +88,7 @@ module.exports.setup = function(cfg) {
     // functions are used as default values for some fields.
     config = jparam(config, _.merge(payload.details, {
       $fromNow: (text) => tc.fromNowJSON(text),
+      timestamp: Math.floor(new Date()),
```
Comment (imbstack) [inline src/intree.js:91]: I'm a js novice, so I'm curious why not `Date.now()`?

**Review by imbstack — APPROVED** [review]

File: src/intree.js (hunk 1 above)
Comment (garbas) [reply src/intree.js:91]: This is the answer stackoverflow gave me :)

**Review by djmitche — APPROVED** [review]

**Pull request merged**